requirements.txt

```diff
@@ -0,0 +1,48 @@
+-abcoll
```

Comment:
What is this file used for?

Reply:
@abose this is auto-generated using dephell
https://dephell.readthedocs.io/installation.html

Reply:
What is dephell used for.

Reply:
This is used to lock the package versions in my current workspace. As I was experimenting with it earlier, I did not create virtualenv and requirement.txt at the start. In order to ensure the smooth working of this package in your system, we need to lock all the package versions which were working for me. Dephell helped me to do it. 

Another alternative would be  ``` pip freeze ``` .  however, I used dephell